src/librustc/infer/region_constraints/mod.rs

```diff
@@ -16,7 +16,7 @@ use self::CombineMapType::*;
 use super::{MiscVariable, RegionVariableOrigin, SubregionOrigin};
 use super::unify_key;
 
-use rustc_data_structures::indexed_vec::IndexVec;
+use rustc_data_structures::indexed_vec::{IndexVec, Idx};
```

Comment:
Stray import?

Reply:
It's needed because methods of Idx trait are used

Reply:
Ah, I see.